scripts/release

```diff
@@ -586,7 +586,9 @@ generate_release_notes() {
     path="$2"
     git diff-tree --no-commit-id --name-only -r "$sha" \
       | grep -v "$path" \
-      | grep -v "MaterialComponents.podspec"
+      | grep -v "MaterialComponents.podspec" \
```

Comment:
Tip: Consider moving this to an array of strings to exclude and a function with a loop?

Reply:
Good suggestion for a follow-up, though it may be better to find a good way to filter out all files in the root of the repo.